modules/graphman-utils.js

```diff
@@ -146,7 +146,7 @@ module.exports = {
             throw "file doesn't exist, " + file;
         }
 
-        return fs.readFileSync(file);
+        return fs.readFileSync(file, { encoding: null });
```

Comment:
when encoding is not specified, it will be read as binary. This seemed correct.

Reply:
If I'm correct, this can be ignored. When we don't specify the encoding or the second parameter, **readFileSync** returns the output as **buffer**.